test/seed.js

```diff
@@ -149,11 +156,53 @@ describe("> Contract: Seed", () => {
             parameterDefaultClass[2]
           );
         });
-        it("should set tipping", async () => {
-          // ToDo in next PR
+        it("should set tip parameters", async () => {
+          /**@type {Tip}*/
+          const tip = await Seed_permissonless.getTip();
+          expect(Seed_permissonless.tipPercentage).to.equal(tip.tipPercentage);
+          expect(Seed_permissonless.tipVestingCliff).to.equal(tip.vestingCliff);
+          expect(Seed_permissonless.tipVestingDuration).to.equal(
+            tip.vestingDuration
+          );
+        });
+        it("should calculate correct totalBuyableAmount", async () => {
+          const hardCap = Seed_permissonless.hardCap;
+          const price = Seed_permissonless.price;
+          const totalBuyableSeed = BigNumber.from(hardCap)
+            .mul(BigNumber.from(PRECISION))
+            .div(BigNumber.from(price));
```

Comment:
Can this be extracted to a helper function.
1. Then add a comment to the helper function pointing to the contract code, that does the `totalBuyableSeed` calculation?

Imo, the contract code should also point to the business logic (if we have it. if not, we should have it)

2. also because this calculation is done multiple times

Reply:
The BL for this again comes from Seed V1. This is currently not in our documentation. I used the calculation in the contracts as basis for this calculation